drakcore/drakcore/frontend/src/AnalysisMain.js

```diff
@@ -27,24 +27,41 @@ class AnalysisMain extends Component {
     }
 
     render() {
-        let processTree = <div>(Process tree was not generated, please check out <kbd>ProcDOT integration (optional)</kbd> section of README to enable it.)</div>;
+        let processTree = <div>(Process tree was not generated, please check out "ProcDOT integration (optional)"
+            section of README to enable it.)</div>;
 
         if (this.state.graph) {
             processTree = <div id="treeWrapper" style={{width: '80em', height: '30em'}}>
-                <Graphviz dot={this.state.graph} />
+                <Graphviz dot={this.state.graph}/>
             </div>;
         }
 
         return <div className="App container-fluid">
-            {processTree}
-
-            <h2>Logs</h2>
-            <div className="list-group">
-            {
-                this.state.logs.map(val => {
-                    return <a href={`/logs/${val.slice(0, -4)}`} class="list-group-item list-group-item-action">{val.slice(37, -4)}</a>
-                })
-            }
+            <div className="page-title-box">
+                <h4 className="page-title">Report</h4>
+            </div>
+
+            <div className="card mb-md-0 mb-3">
+                <div className="card-body">
+                    <h5 className="card-title mb-0">Behavioral graph</h5>
+
+                    {processTree}
+                </div>
+            </div>
+
+            <div className="card mb-md-0 mb-3">
+                <div className="card-body">
+                    <h5 className="card-title mb-0">Analysis logs</h5>
+
+                    <div className="list-group">
+                        {
+                            this.state.logs.map(val => {
+                                return <a href={`/logs/${val.slice(0, -4)}`}
+                                          className="list-group-item list-group-item-action">{val.slice(37, -4)}</a>
```

Comment:
why
`/logs/${val.slice(0, -4)}`}
`{val.slice(37, -4)}` :thinking:  - a comment maybe?`

Reply:
huh, this is pretty strange stuff (not authored by me, tho)

will be fixed